### PR TITLE
chore: Update drivelist to v6.4.2

### PR DESCRIPTION
**Pull request by thundron**
Change-type: patch
Signed-off-by: Lorenzo Alberto Maria Ambrosi <lorenzoa@resin.io>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +58,6 @@
     "chalk": "1.1.3",
     "command-join": "2.0.0",
     "debug": "3.1.0",
-    "drivelist": "6.4.0",
```
Comment (zvin) [inline package.json]: Don't you need to re-add `"drivelist": "6.4.2",` ?

**Comment by jhermsmeier** [comment]
@thundron there's a lint error failing CI:

```
/etcher/lib/start.js
  32:11  error  "../generated/etcher" is not found  node/no-missing-require
```

**Comment by thundron** [comment]
@jhermsmeier I see, it think should be fixed by this https://github.com/resin-io/etcher/pull/2457 but checks have stalled

**Comment by thundron** [comment]
@resin-ci retest

**Comment by thundron** [comment]
@resin-ci retest

**Comment by zvin** [comment]
@resin-ci retest

**Comment by zvin** [comment]
@resin-ci retest

**Pull request merged**